code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -56,6 +56,13 @@
 		for(var/obj/structure/spider/S in range(1, get_turf(src)))
 			return S
 
+/mob/living/simple_animal/hostile/poison/giant_spider/death(gibbed)
+	. = ..()
+	if(!.)
+		return FALSE
+	SSmobs.giant_spiders--
+	return ..()
```

Comment:
This double parent call (there's one a few lines above) seems off

Reply:
Apologies for me not knowing. I'm checking to make sure the spider actually died here and if so we subtract the count from the SSmobs tracker variable. Or am I accidently calling the parent function twice by using `. = ..()` and `return ..()` ?

Reply:
Yes, that is he issue. You need to remove line 64 entirely

Reply:
Thank you, fixing now